pkg/entrypoint/entrypointer.go

```diff
@@ -52,6 +52,22 @@ const (
 	FailOnError     = "stopAndFail"
 )
 
+const (
+	breakpointExitSuffix       = ".breakpointexit"
+	breakpointBeforeStepSuffix = ".beforestepexit"
+)
+
+// DebugBeforeStepError is an error means mark before step breakpoint failure
+type DebugBeforeStepError string
+
+func (e DebugBeforeStepError) Error() string {
+	return string(e)
+}
+
+var (
+	errDebugBeforeStep = DebugBeforeStepError("before step breakpoint error file, user decided to skip the current step execution")
```

Comment:
I'm not sure I understand the meaning of this.
I'm guessing that "an error was encountered while waiting for the before-step breakpoint file because the user decided to skip the execution of the step"? Could you clarify?

Reply:
Maybe it could be rephrased as: "before-step breakpoint: step execution skipped by the user"?